src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -374,6 +374,7 @@ public async Task<JObject> Resolve(ElementAccessExpressionSyntax elementAccess,
                 if (rootObject != null)
                 {
```

Comment:
For a future PR, this block should return early like:
```csharp
if (rootObject is null)
    return null;
```

Code for this method is too nested, which is avoidable.